files/galaxy/tpv/tools.yml

```diff
@@ -1307,6 +1311,8 @@ tools:
   toolshed.g2.bx.psu.edu/repos/iuc/funannotate_annotate/funannotate_annotate/.*:
     cores: 8
     mem: 40
+    params:
+      docker_run_extra_arguments: "--mount type=tmpfs,destination=/tmp"
```

Comment:
Also exclusively set a size for the tmpfs otherwise [50% of hosts mem](https://docs.docker.com/storage/tmpfs/#specify-tmpfs-options) will be used by default